### ISSUE TITLE
launching payara-micro from maven w/ debugging enabled?

### ISSUE THREAD

**Issue by phillipross** [payara/ecosystem-maven#4]
If it's possible, what configuration parameters can be used to launch payara-micro with debugging enabled?  I tried to add jdwp agent command line param to <javaPath> element but plugin didn't like that.

**Comment by mulderbaba** [comment]
@phillipross after getting #31 merged it will be possible to do something like:
```
<systemProperties>
    <property>
        <value>-Xdebug</value>
    </property>
    <property>
        <key>-Xrunjdwp:transport</key>
        <value>dt_socket,server=y,suspend=y,address=5005</value>
    </property>
</systemProperties>
```
I'd suggest to set the ``daemon`` to ``true`` as well :)

**Comment by mulderbaba** [comment]
resolved by #31. @smillidge you can close the ticket.
